Examples/package-info/Sources/package-info/main.swift

```diff
@@ -13,22 +14,16 @@ let packagePath = AbsolutePath(#file).parentDirectory.parentDirectory.parentDire
 
 // There are several levels of information available.
 // Each takes longer to load than the level above it, but provides more detail.
-let diagnostics = DiagnosticsEngine(handlers: [{ print($0)}])
+
+let observability = ObservabilitySystem({ print("\($0): \($1)") })
```

Comment:
Small suggestion: would it make sense to replace with named parameters now that there’s more than one?  I can guess that the first one is the location and the second is the message, but the first might also be severity.  Names would make it clearer in this case.